<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Begin OpenGL</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1" colspan="3">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QSlider" name="z_rot_slider">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Rotate X:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="x_rot_slider">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Rotate Y:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="y_rot_slider">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Rotate Z:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QGLBegin" name="openGLWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QGLBegin</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">qglbegin.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
